src/main/kotlin/lotto/util/RepeatFunction.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.util
+
+import lotto.view.OutputView
+
+object RepeatFunction {
+    fun <T> executeWithRetry(action: () -> T): T {
+        while (true) {
+            try {
+                return action()
+            } catch (e: IllegalArgumentException) {
+                OutputView.printExceptionMessage(e.message)
+            }
+        }
+    }
+}
```

Comment:
```suggestion
    fun <T> executeWithRetry(action: () -> T): T {
        while (true) {
            try {
                return action()
            } catch (e: IllegalArgumentException) {
                OutputView.printExceptionMessage(e.message)
            }
        }
    }
```

Kotlin 에서는 굳이 Util class 이름으로 감싸지 않고 최상단 함수만 선언할 수 있습니다!

그 이유는 자바 바이트 코드로 컴파일 될때 해당 File 이름으로 자동으로 감싸지기 때문이에요 ㅎㅎ